### DISCUSSION TITLE
Compile executables into different directories

### DISCUSSION THREAD

**Discussion by sparky4pro** [nuxui/nuxui#9] [Ideas]
Building a file with nux for macOS will build an *.app file.
Building for iOS does exactly the same.
This means that when compiling for different OS one compilation will override an executable with the same name.

It would be better if nux would create a directory and create the executables in different folders for each target OS.

BTW the in current widgets example the location of buttons and stuff is messed up... on iOS.

Anyway, nuxui seems to be simple to use...

**Comment by nuxart** [comment]
Good idea
